UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Webjob.UnitTests/Services/FulfilmentFileShareServiceTest.cs

```diff
@@ -100,5 +101,58 @@ public void WhenInValidRequest_ThenDownloadFileShareServiceFilesReturnsNoFile()
             var result = fulfilmentFileShareService.DownloadFileShareServiceFiles(message, fulfilmentDataResponses, "");
             Assert.IsNotNull(result);
         }
+
+        [Test]
+        public async Task WhenRequestSearchReadMeFilePath_ThenReturnsFilePath()
```

Comment:
Kindly change WhenRequestSearchReadMeFilePath_ThenReturnsFilePath() to WhenValidSearchReadMeFileRequest_ThenReturnFilePath() and change WhenRequestSearchReadMeFilePath_ThenReturnsEmptyStringResponse() to WhenInvalidSearchReadMeFileRequest_ThenReturnEmptyFilePath()

Reply:
Done